bin/node/runtime/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "node-runtime"
+name = "kitchensink-runtime"
```

Comment:
@kianenigma Do you still want it moved to `bin/kitchensink` or not?

Reply:
no no - Only name should change. I think structure should remain as is

Reply:
I think only name should change and not directory